lstests/doc/README.md

```diff
@@ -467,29 +462,20 @@ center_node_id : int
     Center node ID (for bridge or star topology).
     If topology is bridge, center node ID cannot be 0 or last one.
     No default value.
-tokens_supply : float
-    Total supply of tokens in regular launch mode.
-    Default is 1000000000 (1e9).
 extra_configs : list
-    Extra configs to pass to launcher service.
+    Extra configs to pass to nodeos.
     e.g. ["plugin=SOME_EXTRA_PLUGIN"]
     No default value.
 extra_args : str
     Extra arguments to pass to launcher service.
```

Comment:
Yes. `extra_configs` will be passed via launcher service to `nodeos`.

Reply:
Perhaps `extra_configs` should be named `nodeos_extra_args` and `extra_args` should be named `launcher_extra_args`.

Reply:
There are some new designs with regard to extra arguments. The change is that it is now possible to pass the extra argument from the command line. There are three extra terms now:

- `--extra-service-args` or `Service(extra_args=args)` : Extra arguments to pass to launcher service. It will affect the launcher service itself.
- `--extra-args` or `Cluster(service=serv, extra_args=args)` : Extra arguments to pass to `nodeos` via launcher service. It will affect all the nodes in the cluster. It is one-time argument.
- `--extra-configs` or `Cluster(service=serv, extra_configs=configs)` : Extra configs to pass to `nodeos` via launcher service. It will affect all the nodes in the cluster. It is the configuration that will be appended to the `config.ini`.